agences-regionales-sante/bretagne/2020-03-26.yaml

```
@@ -0,0 +1,28 @@
+date: 2020-03-26
+source:
+    nom: ARS Bretagne
+    url: https://www.bretagne.ars.sante.fr/system/files/2020-03/2020-03-26-bulletin%20information%20COVID-19.pdf
+    archive: https://web.archive.org/web/20200326204531/https://www.bretagne.ars.sante.fr/system/files/2020-03/2020-03-26-bulletin%20information%20COVID-19.pdf
+donneesRegionales:
+  nom: Bretagne
+  code: REG-53
+  casConfirmes: 707 # dont 32 ne residant pas en Bretagne et 61 en cours de rattachement.
+  deces: 29
+  hospitalises: 124
```

```suggestion
  hospitalisesConventionnelle: 124
```
Il me semble qu'il faudrait plutôt utiliser la balise `hospitalisesConventionnelle`. Pour le nombre total d'hospitalisés, pourrait-on donner la somme des conventionnelles+urgences+réanimation+réadaptation ? Ou ce serait une extrapolation à ne pas faire ?